loopable.cpp

```diff
@@ -1,25 +1,12 @@
 #include "loopable.h"
+#include "SmartManager.h"
 
 Loopable::Loopable() {
-    loopables.add(this);
+    SmartManager::addLoopable(this);
```

Comment:
move the adding of Loopables on the `setup()`

Reply:
I'm not sure what you mean. Removing this functionality will remove the whole purpose of Loopable